### DISCUSSION TITLE
Updating from an older V2 to the latest V2 using Quasar CLI and NPM?

### DISCUSSION THREAD

**Discussion by BenJackGill** [quasarframework/quasar#12555] [General - Components / Directives / etc] [answered]
I am updating from an older V2 to the latest V2 using Quasar CLI as per [these docs](https://quasar.dev/start/upgrade-guide#with-quasar-cli).

I have done the Quasar CLI upgrade, but there is also a line in the docs that says "It’s recommended to keep vue & vue-router packages up to date too" and then gives instructions for doing that using Yarn.

But I am using NPM.

What are the matching instructions for NPM?

Or is that not required for NPM?

**Comment by MilosPaunovic** [comment, marked as answer]
`npm i vue@3 vue-router@4`
